modules/ROOT/pages/database-administration/aliases/manage-aliases-composite-databases.adoc

```diff
@@ -0,0 +1,355 @@
+:description: How to use Cypher to manage database aliases in composite databases in Neo4j.
+[role=enterprise-edition aura-db-enterprise]
+[[manage-aliases-composite-databases]]
+= Managing database aliases for composite databases
+
+Both local and remote database aliases can be created as part of a composite database.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE DATABASE `sci-fi-books`;
+CREATE COMPOSITE DATABASE `library`;
+CREATE ALIAS `library`.`sci-fi` FOR DATABASE `sci-fi-books`;
+CREATE COMPOSITE DATABASE garden;
+CREATE DATABASE `perennial-flowers`;
+CREATE ALIAS `library`.`romance` FOR DATABASE `romance-books` AT 'neo4j+s://location:7687' USER alice PASSWORD 'password';
+----
+////
+
+
+[[manage-aliases-composite-databases-list]]
+== List database aliases for composite databases
+
+Available database aliases for composite databases can be seen using `SHOW ALIASES FOR DATABASE`.
+The name of the composite database a particular database alias is targeting appears in the returned `composite` column.
+
+The required privileges are described in the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/dbms-administration/#access-control-dbms-administration-alias-management[Cypher Manual -> Access control -> DBMS administration].
+
+.Query
+[source, cypher]
+----
+SHOW ALIASES FOR DATABASE
+----
+
+.Result
+[role="queryresult"]
+----
++--------------------------------------------------------------------------------------------------+
+| name              | composite | database        | location | url                       | user    |
++--------------------------------------------------------------------------------------------------+
+| "library.romance" | "library" | "romance-books" | "remote" | "neo4j+s://location:7687" | "alice" |
+| "library.sci-fi"  | "library" | "sci-fi-books"  | "local"  | NULL                      | NULL    |
++--------------------------------------------------------------------------------------------------+
+----
+
+For a description of all the returned columns of this command, and for ways in which the `SHOW ALIASES FOR DATABASE` command can be filtered for aliases targeting both standard and composite databases, see xref:database-administration/aliases/manage-aliases-standard-databases.adoc#manage-aliases-list[list aliases for standard databases].
```

Comment:
here we also say aliases targeting composite databases which is how we refer to aliases pointing to the standard databases, I think we need to differ the language slightly when it comes to the aliases belonging to composite databases as they are not aliases pointing to the composite database but instead part of it